fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVJobManager.java

```diff
@@ -213,6 +213,11 @@ public void cancelMTMVTask(CancelMTMVTaskInfo info) throws DdlException, MetaNot
 
     public void onCommit(MTMV mtmv) throws DdlException, JobException {
         MTMVJob job = getJobByMTMV(mtmv);
+        if (!job.getJobStatus().equals(JobStatus.RUNNING)) {
+            LOG.info("job status of async materialized view: [{}] is: [{}], ignore this event.", mtmv.getName(),
```

Comment:
maybe debug log is enough? PAUSED status looks like a very normal case.

Reply:
change to debug